SConscript

```diff
@@ -465,8 +465,8 @@ env.AddMethod(extras, 'PostBuildExtras')
 def buildLibs(projectDir):
     libs_dict = {
         '00std': [                                                              ],
-        '01bsp': [                                                      'libbsp'],
-        '02drv': [                             'libkernel','libdrivers','libbsp'],
+        '01bsp': [                                         'libbsp','libdrivers'],
+        '02drv': [                             'libkernel','libbsp','libdrivers'],
```

Comment:
I'll merge, but please check with @NicolaAccettura for a possible conflict between this an #159.